ansible/roles/cli/tasks/download_build_openwhisk_cli.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+# Download the source code of cli and build localy
+
+- include: docker_login.yml
+
+- shell: "cp -a {{ openwhisk_cli.location }}/bin/ {{ cli_nginx_dir }}"
```

Comment:
This playbook is unfinished, no? Where is the source code checked out? Is it currently assumed that the user does this himself? Is it also assumed the user builds it herself?